File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int[] map = new int[26];
        for(char ch : s.toCharArray()) {
            map[ch-'a']++;
        }
        for(char ch : t.toCharArray()) {
            map[ch-'a']--;
        }
        int count = 0;
        for(int i : map) {
            if(i > 0) count += i;
        }
        return count;
    }
}

File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {number}
 */
var minSteps = function(s, t) {
    let map = new Array(26).fill(0);
    for(let i=0; i<s.length; i++) {
        map[s.charCodeAt(i)-'a'.charCodeAt(0)]++;
        map[t.charCodeAt(i)-'a'.charCodeAt(0)]--;
    }
    let ans = 0;
    for(let count of map) {
        if(count > 0) ans += count;
    }
    return ans;
};